requirements.txt

```diff
@@ -10,3 +10,4 @@ requests>=2.18.4
 bokeh>=0.13.0
 jupyter>=1.0.0
 jupyterlab>=0.35.4
+pyqt5
```

Comment:
I am not sure if adding `pyqt5` here is the best idea. idact will be installed in many places - including cluster nodes. For some clusters pyqt5 may be not available there, also there is no need to have pyqt5 and idact GUI on cluster nodes.
I propose two solutions:
 - move GUI app to a separate python package (thus users would need to type sth like `pip install idactgui` to get it)
 - make conditional checks in the source code, like here:  https://github.com/DataMedSci/pymchelper/blob/master/pymchelper/writers/hdf.py#L21